services/clojars/clojars-version.service.js

```diff
@@ -28,13 +28,19 @@ module.exports = class ClojarsVersion extends BaseJsonService {
   async handle({ clojar }) {
     const json = await this.fetch({ clojar })
 
+    if (!json.latest_version) {
+      throw new InvalidResponse({
+        underlyingError: new Error('version is invalid or null'),
+      })
+    }
+
     if (Object.keys(json).length === 0) {
       /* Note the 'not found' response from clojars is:
           status code = 200, body = {} */
       throw new NotFound()
     }
```

Comment:
A common pattern we like to leverage in service classes is defining a function (usually called `transform`) that handles the responsibility of extracting the desired data point(s) out of the json response. That would be a good update here, especially since this block of code is duplicated in both service classes. 

Let's try to define the `transform` function in a single place (either in a separate/helpers file like `clojars-helpers.js` or create a base class and add it there) and then leverage that function in both classes. 

That would result in the handle function becoming something like:
```js
  async handle({ clojar }) {
    const json = await this.fetch({ clojar })
    const { version } = transform(json)
    return this.constructor.render({ clojar, version })
  }
```

and you can have a transform function somewhere like:
```js
transform(json) {
  if (!json.latest_version) {
    throw new InvalidResponse({
      underlyingError: new Error('version is invalid or null'),
    })
  }

  if (Object.keys(json).length === 0) {
    /* Note the 'not found' response from clojars is:
      status code = 200, body = {} */
    throw new NotFound()
  }

  return { version: json.latest_version }
}
```

Reply:
done!